internal/sessioncmd/terminal.go

```
@@ -179,12 +207,58 @@ func (t *Terminals) Create(sessionID string, opts CreateTerminalOptions) (Termin
+				return runtime.store.CreateSessionBeside(row, caller.ID)
+			}
+		} else {
+			create = func(row store.Session) error {
+				row.ParentID = caller.ID
+				return runtime.store.CreateSession(row)
+			}
+		}
+	}
+	if err := create(sess); err != nil {
+		if killErr := runtime.driver.Kill(sess.ID); killErr != nil {
+			return Terminal{}, fmt.Errorf("%w; its pane %s is still running and has no row: %v", err, sess.ID, killErr)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)